src/App.tsx

```diff
@@ -1,26 +1,105 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import * as todoService from './api/todos';
+import { Todo } from './types/Todo';
+import { Footer } from './components/Footer/Footer';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { ErrorMessages } from './types/Errors';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
 
-const USER_ID = 0;
+type FilterTypes = 'All' | 'Active' | 'Completed';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessages | ''>('');
+  const [selectedFilter, setSelectedFilter] = useState<FilterTypes>('All');
+  const [loading, setLoading] = useState<number[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [disabledButton, setDisabledButton] = useState(true);
+
+  function loadTodos() {
```

Comment:
It is important to follow the same way of creating functions everywhere
```suggestion
  const loadTodos = () => {
```